### ISSUE TITLE
Bundle-RequiredExecutionEnvironment: JavaSE-11 for Jetty 10.0.x

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#3185]
Should we upgrade our OSGi manifests to be 

```
Bundle-RequiredExecutionEnvironment: JavaSE-11
```

Now that Jetty 10.x will be Java 11+ minimum JVM requirement?

**Comment by joakime** [comment]
Eclipse Equinox added support for `JavaSE-11` in bugzilla https://bugs.eclipse.org/532302

**Comment by janbartel** [comment]
If we can find the appropriate artifact on maven central. Currently we have:

```xml 
      <groupId>org.eclipse.platform</groupId>
      <artifactId>org.eclipse.osgi</artifactId>
      <version>3.13.100</version>
```

**Comment by guw** [comment]
@janbartel Do you have a pointer to the OSGi unit tests?

**Comment by janbartel** [comment]
Hi Gunnar, 

The osgi unit tests are here:  https://github.com/eclipse/jetty.project/tree/jetty-10.0.x/jetty-osgi/test-jetty-osgi

To update to JavaSE-11, I changed this line in the jetty project's top level pom and rebuilt: https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/pom.xml#L660

**Comment by guw** [comment]
Are you running the Maven build with JDK 11 or are you using a toolchain.xml? I suspect that the tests will be executed with the Java running the build. Frankly, I don't know if Equinox needed special support for Java 11 execution environment. Most of the bugs I found are related to JDT and Java Development. We have a couple of JDK 11 issues with missing imports because of javax.xml stuff that has been removed.
